unlock-app/src/middlewares/wedlocksMiddleware.ts

```diff
@@ -14,11 +14,9 @@ const wedlocksMiddleware = (config: any) => {
         // existing, so it's safe to simply do nothing in that case.
         if (action.type === SIGNUP_EMAIL && window && window.location) {
           const { origin } = window.location
-          // TODO: then and catch? I think we really only need to worry about errors.
```

Comment:
The `sendEmail` method on `wedlockService` doesn't seem to throw, so I don't know that we can meaningfully do anything about those errors.